packages/uiweb/src/lib/components/space/SpaceWidget/LiveWidgetContent.tsx

```diff
@@ -54,17 +55,40 @@ export const LiveWidgetContent: React.FC<LiveWidgetContentProps> = ({
     }
   };
 
-  useEffect(()=>{
+  const handleEndSpace = async () => {
+    if (!spacesObjectRef?.current) return;
+    await spacesObjectRef?.current?.stop();
+    spacesObjectRef.current = null;
+    setSpaceObjectData(null);
```

Comment:
@samarendra-push I think we should do a `setSpaceObjectData(initSpaceData);` just like how we did while declaring this react state in data provider.

Reply:
Cool